AutomationCSharpTraining/AutomationCSharpTraining/UsernamePage.cs

```diff
@@ -5,34 +5,66 @@
 using System.Threading.Tasks;
 using OpenQA.Selenium;
 using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
 
 namespace AutomationCSharpTraining
 {
     class UsernamePage
     {
         //
         IWebDriver _driver;
-        const string EnterUsernameButtonLocator = "//button[@id = 'passp:sign-in']";
-        const string UsernameFieldLocator = "passp-field-login";
-        
+
+        private static readonly By _enterUsernameButtonLocator = By.XPath("//button[@id = 'passp:sign-in']");
+        private static readonly By _usernameFieldLocator = By.Id("passp-field-login");
+        private static readonly string _username1 = "iifortest";
+        private static readonly string _username2 = "p4vtest";
+
         public IWebElement UsernameField { get; set; }
```

Comment:
Initialize your IWebElement properties straight here.
For example, public IWebElement UsernameField => _driver.FindElement(By.Id(UsernameFieldLocator)). It will allow you to work with web elements at this class level.